unitTests/ui/QuerySuggestAddonTest.ts

```diff
@@ -145,6 +145,48 @@ export function QuerySuggestAddonTest() {
         done();
       });
 
+      it('should receive suggestion when minCharForSuggestion is equal to the length of the text', async done => {
+        const spyOnEvent = jasmine.createSpy('spyOnEvent');
```

Comment:
You are right. The event is already tested in other tests so I'll simply remove it

Reply:
Finally, I will need to have it because this is what will tell me if I showed suggestions to the users. I would let it inside my current test cases because I want to know if I get a suggestion when I am lower, equal or higher then the length set in the option.

Reply:
Hmm, I see. Is that the only way, because it is a little indirect? If we look at the source code:

```
if (text.length >= this.omnibox.options.minCharForSuggestions) {
    return this.cache.getSuggestions(text, () => this.getQuerySuggest(text));
}
```
Is it possible to mock the `getSuggestions` function on `this.cache`? If not then I guess the event is the best we can do.

Reply:
the cache is private so to mock it, I would need to either put it public or create accessor to the variable.
Do you think it's worth it or should we continue to use the current event?

Reply:
The event is fine in that case, thanks for investigating.